actions/pr-comment/action.yml

```
@@ -0,0 +1,15 @@
+name: "PR Comment Action"
+description: "Posts a comment on pull requests."
+
+inputs:
+  github-token:
+    description: 'The `GITHUB_TOKEN` secret used to post comments on pull requests. If not specified, the automatic GitHub token provided to the workflow will be used.'
+    required: false
```

```suggestion
    required: true
```
Suggestions work too